app/Services/DatatableService.php

```diff
@@ -51,7 +51,13 @@ public function query() {
 			$sort = explode('|', $this->request->input('sort'));
 			$query->orderBy($sort[0], $sort[1]);
 		} else {
-			$query->orderBy("{$tableName}.created_at", 'desc');
+			if (isset($this->queryConfig['table'])) {
```

Comment:
this is not ok - what if someone wants to use table somewhere else? you can add an order by maybe, or initialize the table to start ordering by it